tests/dtm/test_single_process_restart.py

```diff
@@ -84,6 +86,7 @@ def setup_class(cls):
                                 cls.master_node_list[0].password)
         cls.test_cfg = configmanager.get_config_wrapper(fpath="config/test_dtm_config.yaml")
         cls.m0d_process = 'm0d'
+        cls.rwg_process = 'rgw'
```

Comment:
@priyankaborawake  as discussed, i am creating different file for rgw tests.
For now we can keep Rohan's tests in this fil only and port tests once my PR is merged.

Reply:
Sure, we will port this test.